syntax = "proto3";

package protos;
option go_package = "./proto";
import "protos/message.proto";
// 定义交易履约系统服务接口
service Cpp_C {
  rpc Cpp_C_1(CommonMessage) returns (CommonMessage) {}
  rpc Cpp_C_2(CommonMessage) returns (CommonMessage) {}
  rpc Cpp_C_3(CommonMessage) returns (CommonMessage) {}
}
